packages/ruby/lib/readme/errors.rb

```diff
@@ -0,0 +1,35 @@
+module Readme
+  module Errors
+    API_KEY_ERROR = "Missing API Key"
+    REJECT_PARAMS_ERROR = "reject_params option must be an array of strings"
+    ALLOW_ONLY_ERROR = "allow_only option must be an array of strings"
+    BUFFER_LENGTH_ERROR = "buffer_length must be an Integer"
+    DEVELOPMENT_ERROR = "development option must be a boolean"
+
+    MISSING_BLOCK_ERROR = <<~MESSAGE
+      Missing block argument. You must provide a block when configuring the
+      middleware. The block must return a hash containing user info:
+        use Readme::Metrics, options do |env|
+          { id: "unique_id", label: "Your user label", email: "Your user email" }
+        end
+    MESSAGE
+
+    class BadBlockMessage
+      def self.call(result)
```

Comment:
Thoughts on making this a class method `Readme::Errors.bad_block_message` rather than a callable constant? I think that might be a bit more idiomatic.